src/handlers/farmerHandlers.js

```diff
@@ -0,0 +1,106 @@
+import axios from 'axios';
+import { pathObj } from '../utils/generalVariables';
+
+export const getFarmersHandler = (token) => {
+
+  return axios.get(`${pathObj.getFarmersPath}`, { 
+    headers: {
```

Comment:
Setting the headers in every request could be dangerous havint it in multiple places. A better approach is to have it set up in a single place, something like this https://stackoverflow.com/a/51445755/11761341

Reply:
I agree, alternatively just set it as a global const. to prevent redundancy.

Reply:
That's pretty much what I did ☝️ 

I made a basic utility function that takes the token as a parameter and simply returns the headers object, which can just be plopped down in the Axios request. This was the fastest fix, in my opinion